dbt_project.yml

```diff
@@ -1,10 +1,11 @@
 name: 'ad_reporting'
-version: '0.8.0'
+version: '1.0.0'
 config-version: 2
 
 require-dbt-version: [">=1.0.0", "<2.0.0"]
 
 vars:
+  ad_reporting__apple_search_ads_enabled: true
```

Comment:
@fivetran-jamie actually uncovered that we should remove these and rely on the inline default values as they cause conflicts in the source. Can you remove these variables.